diff --git a/chips/sf32lb52/Kconfig b/chips/sf32lb52/Kconfig
index 11fa808..a256c69 100644
--- a/chips/sf32lb52/Kconfig
+++ b/chips/sf32lb52/Kconfig
@@ -20,6 +20,7 @@ config ARCH_CHIP_SF32LB52
 	select ARCH_HAVE_FETCHADD
 	select ARCH_HAVE_HEAPCHECK
 	select ARCH_HAVE_PROGMEM
+	select ARCH_HAVE_SERIAL_TERMIOS
 	select ARCH_HAVE_SPI_BITORDER
 	select ARM_HAVE_MPU_UNIFIED
 	select ARMV8M_HAVE_STACKCHECK
